.github/workflows/ci.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -e .[all]
+          pip install -e .[all] 
```

Comment:
Unnecessary space at end of text (which is auto-deleted in Sublime/VSCode/Pycharm unless the user configures it to disable that, and then it appears in next PRs)